### DISCUSSION TITLE
Agent Pregens

### DISCUSSION THREAD

**Discussion by ericcberg** [deltagreen-foundryvtt/delta-green-foundry-vtt-system#103] [Ideas]
Can the agent pregens get sorted into folders by profession? Having them all in one folder is rather unwieldy.

**Comment by jalensailin** [comment]
@TheLastScrub this could be included as part of the necessary pregen fixes

**Comment by TheLastScrub** [comment]
Yep I was thinking they could use a sorting after I realized that was actually possible looking at other systems. Considering I'll have to re-import them anyways this would make sense to do at the same time I think.
